modules/sdcore-user-plane-k8s/outputs.tf

```diff
@@ -0,0 +1,12 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+output "upf_app_name" {
```

Comment:
Same as above. I think we should also have available endpoints here.

Reply:
Done.